crates/graphql-composition/src/subgraphs/extensions.rs

```diff
@@ -9,10 +9,6 @@ pub(crate) struct ExtensionRecord {
 }
 
 impl Subgraphs {
-    pub(crate) fn extension_is_defined(&self, name: StringId) -> bool {
```

Comment:
Why removed?

Reply:
The duplicate extensions check is pushed back further. I'm still not 100% happy and settled with a solution for deduplicating extensions from the subgraphs.